hugo/themes/dora-team/theme.toml

```diff
@@ -0,0 +1,20 @@
+# theme.toml template for a Hugo theme
+# See https://github.com/gohugoio/hugoThemes#themetoml for an example
+
+name = "dora-team"
+license = "CC BY-NC-SA 4.0 DEED"
+licenselink = "https://creativecommons.org/licenses/by-nc-sa/4.0/"
+description = ""
+homepage = ""
+tags = []
+features = []
+min_version = "0.41.0"
```

Comment:
Is this the minimum Hugo version required? I think we should go with something much newer. IIRC, anything less than, like 0.112 fails. Also, it needs `extended` the SCSS processing. IDK how one specifies that in this TOML file though.

Reply:
This was auto-created base upon my version of hugo (`hugo v0.106.0`).  Not sure the details on why "they" chose this version, but I can update

Reply:
We can add "extended" in the version name (I believe).  

Do you want to start with version as defined in the Docker image.  Currently, ENV HUGO_VERSION=0.114.1

Reply:
Does not look like this is a technical constraint, but only for people consumption.

Updated to Docker version